Tasks/UseDotNetV2/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 184,
-        "Patch": 0
+        "Patch": 1
```

Comment:
Could you please remove version bump? There are changes only for md file - so it won't affect task.